### DISCUSSION TITLE
Weather 2 - Remastered Is Live On Curseforge!

### DISCUSSION THREAD

**Discussion by Mrbt0907** [CosmicHikari/Weather2-Remastered#107] [Announcements]
Hello everyone! I am happy to announce that the current 2.7 update is now live on Curseforge! All further updates will now be released here and there, making things easier for you guys and me. You can find the project here: https://www.curseforge.com/minecraft/mc-mods/weather-2-remastered

I hope I can finish 2.8 before the end of this year for you guys.

Have a great day everyone!

**Comment by juraj-hrivnak** [comment]
That is awesome to hear! Thank you for your effort!

**Reply by Mrbt0907** [reply]
Cheers! I hope to get a lot of things done this week

**Comment by Desoroxxx** [comment]
I will now contact EnderLanky for possible compatibility

**Reply by Mrbt0907** [reply]
I would wait for update 2.8 as the api would be flushed out then.

**Comment by agentwolf88** [comment]
Will you port to 1.18.2?

**Reply by Mrbt0907** [reply]
Not in the near future.

**Comment by CosmicHikari** [comment]
Not as of yet. There are many plans for the 1.12.2 branch still :)

**Comment by Desoroxxx** [comment]
Quick question you removed the April's fool chicken, right?

**Reply by Mrbt0907** [reply]
Nope, the april fools joke is still there. Now that you reminded me of that, I should add more content like that...

**Reply by Desoroxxx** [reply]
Yeah no, the chicken is always loaded in memory and thus takes a ton more ram/vram it takes twice as much as the entirety of vanilla entire atlas

**Reply by Desoroxxx** [reply]
So, if you want to add more content like that/keep the chicken you need to find a way to make it configurable and either disable it by default or make it only load when the day is 1 of April

**Reply by Desoroxxx** [reply]
Also, it takes so much space on the atlas that it may make some modpack unable to add Weather 2 since the current 1.12.2 altas generation code is limited in size

**Reply by Mrbt0907** [reply]
Do you have proof of this? I don't notice much of a difference with it unloaded.

**Reply by Mrbt0907** [reply]
If that were true, then the other 256x256 textures should be overloading minecraft since the majority of particle textures are the same size.

**Reply by Desoroxxx** [reply]
Well yes, they are but at least they serve a purpose

![image](https://user-images.githubusercontent.com/82710983/201675272-fc8e0d8d-e73a-4f3a-bb87-ed9285e81d8e.png)

Another options you have probably the best is doing your own atlas, Forestry does it

**Reply by Desoroxxx** [reply]
In my opinion you should go for making your own Atlas it would both increase performance and give you the most control and also the most possibility no downside to adding a chicken or anything

**Reply by Desoroxxx** [reply]
I would like to help with this mod do you know when you will push 2.8 and get GitHub up to date, I could help with optimization it's what I love doing

**Reply by Mrbt0907** [reply]
I'll see what I can do. When I get the next indev done, the code will be posted here. I am about 75% done with that I believe.

**Comment by CosmicHikari** [comment]
We've got cows!
